### ISSUE TITLE
Sightings are difficult to send out independently

### ISSUE THREAD

**Issue by terrymacdonald** [johnwunder/twigs#13]
### PROBLEM

In the current version of STIX Sightings are defined as Sightings of an Indicator. They are tied very closely to the Indicator, and are most often embedded within the Indicator. This close tie creates some problems which restrict the usefulness of Sightings:
- In order to add a Sighting to an Indicator the producer of the original Indicator would need to publish a new version of that Indicator with the new Sightings added.
- If a third-party wants to inform the producer that they have a Sighting related to the original Indicator there is no way to do so. The third-party will need to send the original producer a new Indicator with the similar details in it, but under the third-parties namespace, and then attach the Sightings they have to their own Indicator object. The producer of the original Indicator would need to somehow recognize that the Indicator is the similar to the same one that they sent out earlier, and would need to resend a new updated original Indicator with the third-parties Sightings attached.
### POTENTIAL ANSWER

This proposal requires the changes suggested within section 5 – “Observables, Observable Patterns and Observable Instances differences aren’t easily discerned”. In that proposal the name of Observable Instances (STIX Observables) would be changed to become STIX ‘Observations’. STIX Observations would be restricted from use within the STIX Indicator object. 

Observable Patterns would be re-labeled STIX ‘Patterns’. STIX Patterns would only be allowed to live within the STIX Indicator Object, restricting them to describing what one would need to look for in order for the Indicator to trigger. 

This separation of function would make the role of the Indicator and Sighting easier to understand for new users of STIX; The Indicator contains ‘things you should look for’, and the Observation contains ‘things you’ve seen’.
#### Proposal
- Rename Observable Instances to STIX Observations
- Rename Observable Patterns to STIX Patterns
- The new top-level Relationship Object (mentioned elsewhere in this document) is used to relate that fact a Sighting was detected. The relationship would link the Observation with the Indicator with a relationship type of ‘Sighting_of’ or something similar. 
- Observable patterns aren’t used within an Observation object, and instead live within an Indicator.
- The Indicator object is also restricted to only containing STIX Patterns, and loses the ability to contain Observations.
- Deprecate embedded indicator sightings.

This effectively creates a nice separation between 'things we need to look for' (Indicators+ Patterns) and 'things we have found' (Observations).

**Comment by johnwunder** [comment]
Issue 306, Roadmap 1 in STIX tracker

**Comment by terrymacdonald** [comment]
Within TWIGS we have a separate 'Relationship Object'. This object is used to allow relationships to be sent independently of the Objects that they refer to. This allows for third-parties to create relationships between objects that others have disseminated previously. 

I consider a sighting to be a relationship between an Observable, and an Indicator. The Indicator tells an Organization what to look for, and the when the Indicator is detected, the Organization is able to generate a Relationship object and (if they want to) an Observation Object where the Observation Object describes what they detected, and the Relationship Object links the two together.

**Comment by terrymacdonald** [comment]
Another point to make is that we should allow relationships to exist between any STIX Domain objects. The goal is to provide enough flexibility for the communities to decide what makes sense to them about how to relate the top-level Domain objects together. It has always confused me as to why certain relationships are allowed within the current version of STIX, but that some others are not. I find it strange that there are some seemingly arbitrary restrictions on what thing can relate to what other thing. By providing a separate relationship object we are providing a framework that can apply to any top-level object within STIX, so we should allow communities the full STIX palette with which to describe what they are seeing.

In other words - the restriction of what can be described should always be what the producer is comfortable with disseminating - not what restrictions have been placed on them via the model.

**Comment by terrymacdonald** [comment]
Added to 'Why TWIGS is TWIGS' documentation.
